src/libraries/Microsoft.PowerFx.Connectors/OpenApiExtensions.cs

```diff
@@ -340,7 +340,7 @@ internal static bool TryGetOpenApiValue(IOpenApiAny openApiAny, FormulaType form
             }
             else if (openApiAny is IDictionary<string, IOpenApiAny> o)
             {
-                Dictionary<string, FormulaValue> dvParams = new ();
+                Dictionary<string, FormulaValue> dvParams = new();

```

Comment:
>Dictionary<string, FormulaValue> dvParams = new(); [](http://example.com/codeflow?start=16&length=50)

If there is no actual change, better to avoid this spacing change. This also applies to lines 987, 1161, 1190, 1223, 1320, 1352.